build/lib/simple_create_tarballs.sh

```diff
@@ -25,8 +25,8 @@ BINARY_PLATFORMS="$5"
 TAR_PATH="$6"
 GIT_HASH="$7"
 
-LICENSES_PATH="$OUTPUT_DIR/LICENSES"
-ATTRIBUTION_PATH="$OUTPUT_DIR/ATTRIBUTION.txt"
+LICENSES_PATHS=($(find $OUTPUT_DIR -type d -name "LICENSES"))
```

Comment:
oh dang im surprised this wasnt already handled. i guess we just didnt notice for projects with multiple attr files they werent getting put in the tarball?  or maybe this is the first one that produces a tarball and multiple attr files?

Reply:
Yes even i was surprised. There are others like the tinkerbell actions that produce multiple attribution files but we don't have tarball for those